test/library/packages/UnitTest/Launcher.chpl

```diff
@@ -15,6 +15,32 @@ module Launcher {
   config const setComm: string = "";
   var comm: string;
 
+  /* Gets the comm */
+  proc getRuntimeComm() throws {
+    var line: string;
+    var checkComm = spawn(["python",CHPL_HOME:string+"/util/chplenv/chpl_comm.py"],
```

Comment:
I wonder if it will matter if the user is working with a *different* chapel installation than the one that compiled the launcher, i.e. `runtime CHPL_HOME != compile-time CHPL_HOME`.

Reply:
No, it doesn't matter where you compile the launcher as the use of `comm` is just to verify that you are in a multilocale mode.